extendr-api/src/rmacros.rs

```diff
@@ -144,7 +144,7 @@ macro_rules! rprint {
     () => {
     };
     ($($rest: tt)*) => {
-        print_r_output(format!($($rest)*));
+        print_r_output(format!($($rest)*)).replace("%","%%");
```

Comment:
Isn't the way to escape these is to use `.replace("%", "\%")`?

Reply:
fair question

[this reference ](https://en.cppreference.com/w/cpp/io/c/fprintf) says it should be `%%`, [this SO says that](https://stackoverflow.com/questions/20583827/difference-between-and) `\%` is illegal because it does not have any defined meaning.

printf uses a literal `%` as the only special character
the escaped `%` as `\%` does not have a defined meaning and should be avoided
printf uses a `%` the special character to escape `%` when needed literally, therefore `%%`

Reply:
No, you're right about the `%%`.